src/error.js

```diff
@@ -153,6 +153,17 @@ export function cancellation() {
 }
 
 /**
+ * @param {*} errorOrMessage
+ * @return {boolean}
+ */
+export function isCancellation(errorOrMessage) {
+  if (!errorOrMessage) {
+    return false;
+  }
+  return errorOrMessage == CANCELLED || errorOrMessage.message == CANCELLED;
```

Comment:
any reason why `Error.cancellation()` returns new instance everytime, instead of using a global var?